src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        int index = new Random().nextInt(Color.values().length);
+        Color color = Color.values()[index];
+        return String.valueOf(color);
```

Comment:
Instead of using `String.valueOf(color)`, it's better to use `color.name()` to get the String representation of the enum constant, as per the checklist.

Reply:
Instead of using String.valueOf(color), it's better to use color.name() to get the String representation of the enum constant, as per the checklist.